debug/examples/shop-consumables-confirm.json5

```diff
@@ -30,12 +30,18 @@
         align: "center",
     },
     confirm: {
+        audio: {
+            buy: "buy",
+            equip: "equip",
+            unequip: "unequip",
+            use: "use"
+        },
```

Comment:
do we need this as config or can we just reserve these keys for audio?

Reply:
Prob best to let them configure the key since they might not be using the shop and then config this anyway?